utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -192,6 +207,203 @@ Describe 'File/folder tests' -Tag 'Modules' {
+        It '[<moduleFolderName>] Module must contain a [` CHANGELOG.md `] file.' -TestCases ($moduleFolderTestCases | Where-Object { $_.moduleVersionExists }) {
+
+            param(
+                [string] $moduleFolderPath,
+                [bool] $moduleVersionExists
+            )
+
+            $pathExisting = Test-Path $changelogFilePath
+            $pathExisting | Should -Be $true
+        }
+
+        It '[<moduleFolderName>] `CHANGELOG.md` must not be empty.' -TestCases ($moduleFolderTestCases | Where-Object { $_.moduleVersionExists }) {
+
+            param(
+                [string] $moduleFolderPath,
+                [bool] $moduleVersionExists
+            )
+
+            if (-not (Test-Path -Path $changelogFilePath)) {
+                Set-ItResult -Skipped -Because 'CHANGELOG.md file not found.'
+                return
+            }
+
+            # the changelog should not be empty
+            $changelogContent.Length | Should -BeGreaterThan 0 -Because 'the changelog should not be empty'
+        }
+
+        It '[<moduleFolderName>] `CHANGELOG.md` must start with `#Changelog`, an empty line and a link to the latest version.' -TestCases ($moduleFolderTestCases | Where-Object { $_.moduleVersionExists }) {
+
+            param(
+                [string] $moduleType,
+                [string] $moduleFolderName
+            )
+
+            if (-not (Test-Path -Path $changelogFilePath) -or $changelogContent.Count -le 4) {
+                Set-ItResult -Skipped -Because 'CHANGELOG.md file not found or uncomplete.'
+                return
+            }
+
+            # the changelog should start with '# Changelog'
+            $changelogContent[0] + $changelogContent[1] | Should -Be '# Changelog' -Because 'the changelog should start with `# Changelog`, followed by an empty line'
+
+            # The latest version of the changelog can be found [here](/Azure/bicep-registry-modules/blob/main/avm/res/aad/domain-service/CHANGELOG.md).
+            $linkToLatestVersionChangelog = 'The latest version of the changelog can be found [here](/Azure/bicep-registry-modules/blob/main/avm/{0}/{1}/Changelog.md).' -f $moduleType, $moduleFolderName
+            $changelogContent[2] + $changelogContent[3] | Should -Be $linkToLatestVersionChangelog -Because ('the changelog must contain a link to the latest version {0}' -f $linkToLatestVersionChangelog)
+        }
+
+        It '[<moduleFolderName>] `CHANGELOG.md` must contain a section for the module version.' -TestCases ($moduleFolderTestCases | Where-Object { $_.moduleVersionExists }) {
+
+            param(
+                [string] $moduleType,
+                [string] $moduleFolderName
+            )
+
+            if (-not (Test-Path -Path $changelogFilePath) -or $changelogContent.Count -le 4) {
+                Set-ItResult -Skipped -Because 'CHANGELOG.md file not found or uncomplete.'
+                return
+            }
```

Comment:
```suggestion
            if (-not (Test-Path -Path $changelogFilePath)) {
                Set-ItResult -Skipped -Because "CHANGELOG.md file was not found in path [$changelogFilePath]."
                return
            }
            if ($changelogContent.Count -le 4) {
                Set-ItResult -Skipped -Because "The CHANGELOG.md file's header is either missing or incorrect. You can find an example of the header [here](https://azure.github.io/Azure-Verified-Modules/spec/BCPNRF22/#example-content-of-the-changelogmd)."
                return
            }
```

Note @ReneHezser, this header does not exist as required by these tests here in the AVM specs. See the link.